targets/TARGET_RENESAS/TARGET_RZ_A1XX/sleep.c

```diff
@@ -0,0 +1,196 @@
+                if ((*p_module->p_stback & p_module->stbrq) == 0) {
+                    break;
+                }
+            }
+        }
+        p_module++;
+    }
+    (void)dummy_8;
+}
+
+void hal_sleep(void) {
+    // Transition to Sleep Mode
+    __WFI();
+}
+
+void hal_deepsleep(void) {
+    volatile uint8_t dummy_8;
+
+    core_util_critical_section_enter();
+    wk_CPGSTBCR3  = CPGSTBCR3;
+    wk_CPGSTBCR4  = CPGSTBCR4;
+    wk_CPGSTBCR5  = CPGSTBCR5;
+    wk_CPGSTBCR6  = CPGSTBCR6;
+    wk_CPGSTBCR7  = CPGSTBCR7;
+    wk_CPGSTBCR8  = CPGSTBCR8;
+    wk_CPGSTBCR9  = CPGSTBCR9;
+    wk_CPGSTBCR10 = CPGSTBCR10;
+    wk_CPGSTBCR11 = CPGSTBCR11;
+    wk_CPGSTBCR12 = CPGSTBCR12;
+#if defined(TARGET_RZA1H)  || defined(TARGET_VK_RZ_A1H)
+    wk_CPGSTBCR13 = CPGSTBCR13;
+#endif
+
+    /* MTU2 (for low power ticker) */
+    CPGSTBCR3  |= ~(CPG_STBCR3_BIT_MSTP33);
+    dummy_8    = CPGSTBCR3;
+    CPGSTBCR4  = 0xFF;
+    dummy_8    = CPGSTBCR4;
+    CPGSTBCR5  = 0xFF;
+    dummy_8    = CPGSTBCR5;
+    /* Realtime Clock, JCU, CEU */
+    CPGSTBCR6  |= ~(CPG_STBCR6_BIT_MSTP60 | CPG_STBCR6_BIT_MSTP61 | CPG_STBCR6_BIT_MSTP66);
+    dummy_8    = CPGSTBCR6;
+    /* Video Decoder0, Video Decoder1, Ethernet */
+    CPGSTBCR7  |= ~(CPG_STBCR7_BIT_MSTP77 | CPG_STBCR7_BIT_MSTP76 | CPG_STBCR7_BIT_MSTP74);
+    dummy_8    = CPGSTBCR7;
+    /* EthernetAVB, MediaLB, IMR-LSD, IMR-LS2_1, IMR-LS2_0 */
+    CPGSTBCR8  |= ~(CPG_STBCR8_BIT_MSTP82 | CPG_STBCR8_BIT_MSTP83 | CPG_STBCR8_BIT_MSTP85 | CPG_STBCR8_BIT_MSTP86 | CPG_STBCR8_BIT_MSTP87);
+    dummy_8    = CPGSTBCR8;
+    /* VDC5_1, SPI Multi I/O Bus Controller0 */
+    CPGSTBCR9  |= ~(CPG_STBCR9_BIT_MSTP90 | CPG_STBCR9_BIT_MSTP93);
+    dummy_8    = CPGSTBCR9;
+    /* OpenVG */
+    CPGSTBCR10 |= ~(CPG_STBCR10_BIT_MSTP100);
+    dummy_8    = CPGSTBCR10;
+    CPGSTBCR11 = 0xFF;
+    dummy_8    = CPGSTBCR11;
+    CPGSTBCR12 = 0xFF;
+    dummy_8    = CPGSTBCR12;
+#if defined(TARGET_RZA1H)  || defined(TARGET_VK_RZ_A1H)
+    CPGSTBCR13 = 0xFF;
+    dummy_8    = CPGSTBCR13;
+#endif
+
+    module_standby_in();
+
+    // Transition to Sleep Mode
+    __WFI();
+
+    CPGSTBCR3  = wk_CPGSTBCR3;
+    dummy_8    = CPGSTBCR3;
+    CPGSTBCR4  = wk_CPGSTBCR4;
+    dummy_8    = CPGSTBCR4;
+    CPGSTBCR5  = wk_CPGSTBCR5;
+    dummy_8    = CPGSTBCR5;
+    CPGSTBCR6  = wk_CPGSTBCR6;
+    dummy_8    = CPGSTBCR6;
+    CPGSTBCR7  = wk_CPGSTBCR7;
+    dummy_8    = CPGSTBCR7;
+    CPGSTBCR8  = wk_CPGSTBCR8;
+    dummy_8    = CPGSTBCR8;
+    CPGSTBCR9  = wk_CPGSTBCR9;
+    dummy_8    = CPGSTBCR9;
+    CPGSTBCR10 = wk_CPGSTBCR10;
+    dummy_8    = CPGSTBCR10;
+    CPGSTBCR11 = wk_CPGSTBCR11;
+    dummy_8    = CPGSTBCR11;
+    CPGSTBCR12 = wk_CPGSTBCR12;
+    dummy_8    = CPGSTBCR12;
+#if defined(TARGET_RZA1H)  || defined(TARGET_VK_RZ_A1H)
+    CPGSTBCR13 = wk_CPGSTBCR13;
+    dummy_8    = CPGSTBCR13;
+#endif
+
+    module_standby_out();
+    core_util_critical_section_exit();
+
+    (void)dummy_8;
```

Comment:
specific meaning to do this dummy8 after waking up?

Reply:
It is intended to implicitly avoid warnings to unused variables and has no particular meaning.